src/api/app/controllers/webui/project_controller.rb

```diff
@@ -193,13 +193,7 @@ def new_release_request
   def show
     @bugowners_mail = @project.bugowner_emails
 
-    # An incident has a patchinfo if there is a package 'patchinfo' with file '_patchinfo', try to find that:
-    @has_patchinfo = false
-    if @packages.map { |p| p[0] }.include?('patchinfo')
-      Directory.hashed(project: @project.name, package: 'patchinfo').elements('entry') do |e|
-        @has_patchinfo = true if e['name'] == '_patchinfo'
-      end
-    end
+    @has_patchinfo = PackageKind.where(package_id: @project.package_ids, kind: 'patchinfo').exists?
```

Comment:
nice catch @bgeuken !

Reply:
Probably, but we would still store this in a variable since it's used multiple times in the UI. So not sure we would gain much.